source/lac/trilinos_epetra_vector.cc

```diff
@@ -170,17 +170,20 @@ namespace LinearAlgebra
       const Vector &down_V = dynamic_cast<const Vector &>(V);
       // If the maps are the same we can Update right away.
       if (vector->Map().SameAs(down_V.trilinos_vector().Map()))
-        vector->Update(1., down_V.trilinos_vector(), 1.);
+        {
+          const ierr = vector->Update(1., down_V.trilinos_vector(), 1.);
+          Assert(ierr==0, ExcTrilinosError(ierr));
+        }
       else
         {
           Assert(this->size()==down_V.size(),
                  ExcDimensionMismatch(this->size(), down_V.size()));
 
 #if DEAL_II_TRILINOS_VERSION_GTE(11,11,0)
           Epetra_Import data_exchange (vector->Map(), down_V.trilinos_vector().Map());
-          int ierr = vector->Import(down_V.trilinos_vector(), data_exchange, Epetra_AddLocalAlso);
+          const int ierr = vector->Import(down_V.trilinos_vector(),
+                                          data_exchange, Epetra_AddLocalAlso);
           Assert(ierr==0, ExcTrilinosError(ierr));
-          (void) ierr;
```

Comment:
Doesn't deleting this lead to warnings of the form `variable ierr is set but not used`? Or why should it no longer be necessary?